modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/roundstartslime.dm

```diff
@@ -33,212 +51,433 @@
+			var/obj/item/organ/got_organ = alterer.getorganslot(slot)
+			if(got_organ)
+				got_organ.Remove(alterer)
+				qdel(got_organ)
+		else
+			alterer.dna.species.mutant_bodyparts -= chosen_key
+	else
+		if(selected_sprite_accessory.organ_type)
+			var/obj/item/organ/organ_path = selected_sprite_accessory.organ_type
+			var/slot = initial(organ_path.slot)
+			var/obj/item/organ/got_organ = alterer.getorganslot(slot)
+			if(got_organ)
+				got_organ.Remove(alterer)
+				qdel(got_organ)
+			organ_path = new selected_sprite_accessory.organ_type
+			var/list/new_acc_list = list()
+			new_acc_list[MUTANT_INDEX_NAME] = selected_sprite_accessory.name
+			new_acc_list[MUTANT_INDEX_COLOR_LIST] = selected_sprite_accessory.get_default_color(alterer.dna.features, alterer.dna.species)
+			alterer.dna.mutant_bodyparts[chosen_key] = new_acc_list.Copy()
+			if(ROBOTIC_DNA_ORGANS in alterer.dna.species.species_traits)
+				organ_path.status = ORGAN_ROBOTIC
+				organ_path.organ_flags |= ORGAN_SYNTHETIC
+			organ_path.build_from_dna(alterer.dna, chosen_key)
+			organ_path.Insert(alterer, 0, FALSE)
+
+		else
+			var/list/new_acc_list = list()
+			new_acc_list[MUTANT_INDEX_NAME] = selected_sprite_accessory.name
+			new_acc_list[MUTANT_INDEX_COLOR_LIST] = selected_sprite_accessory.get_default_color(alterer.dna.features, alterer.dna.species)
+			alterer.dna.species.mutant_bodyparts[chosen_key] = new_acc_list
+			alterer.dna.mutant_bodyparts[chosen_key] = new_acc_list.Copy()
+		alterer.dna.update_uf_block(GLOB.dna_mutant_bodypart_blocks[chosen_key])
+	alterer.update_mutant_bodyparts()
+
+/**
+ * Alter markings lets you add a particular body marking
+ */
+/datum/action/innate/alter_form/proc/alter_markings(mob/living/carbon/human/alterer)
+	var/list/candidates = GLOB.body_marking_sets
+	var/chosen_name = tgui_input_list(
+		alterer,
+		"Select which set of markings would you like to change into",
+		"Marking Alterations",
+		candidates,
+	)
+	if(!chosen_name)
+		return
+	var/datum/body_marking_set/marking_set = GLOB.body_marking_sets[chosen_name]
+	alterer.dna.species.body_markings = assemble_body_markings_from_set(marking_set, alterer.dna.features, alterer.dna.species)
+	alterer.update_body(is_creating = TRUE)
+
+/**
+ * Alter genitals lets you adjust the size or functionality of genitalia
+ * If you don't own the genital you try to adjust, it'll ask you if you want to add it first
+ */
+/datum/action/innate/alter_form/proc/alter_genitals(mob/living/carbon/human/alterer)
+	var/dna_alteration = tgui_input_list(
+		alterer,
+		"Select what bodypart you'd like to alter",
+		"Genitalia Alteration",
+		list(
+			"Breasts Lactation",
+			"Breasts Size",
+			"Penis Girth",
+			"Penis Sheath",
+			"Penis Length",
+			"Penis Taur Mode",
+			"Testicles Size",
+			),
+	)
+	if(!dna_alteration)
+		return
+	switch(dna_alteration)
+		if("Breasts Lactation")
+			if(!alterer.getorganslot(ORGAN_SLOT_BREASTS))
+				alterer.balloon_alert(alterer, "no breasts!")
```

Comment:
if you do the early check for body parts, this could be removed. I won't go through on each one, but yea.